packages/fluentui/react-northstar/src/utils/htmlPropsUtils.tsx

```diff
@@ -156,10 +156,13 @@ export const htmlInputEvents: HtmlInputEvents[] = [
 ];
 
 export const htmlInputProps: HtmlInputProps[] = [...htmlInputAttrs, ...htmlInputEvents];
+export interface htmlCustomProps {
+  [index: number]: string;
+}
```

Comment:
What is this for ?

Reply:
I wanted to separate(define) only my "custom" aria props.
But on the line 165 there was "htmlProps" typed only to HtmlInputProps[], which I don't need. It cause issue, that I got together with my aria props as well "keydown" object/event , and it broke some other behavior, then.
So I added this type to be able set any custom props which I need.
Not sure if this is correct way. Hope so :)

Reply:
I've added a suggestion to your usage of `partitionHtmlProps`, you shouldn't need this


also this is just an array in disguise :) there is no difference between this and `string[]`

Reply:
As we discussed offline, just add `AriaAttributes` type in a union